moped-database/migrations/1608670107519_alter_table_public_moped_project_add_column_project_geometries/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "public"."moped_project" ADD COLUMN "project_extent_geojson" json NULL;
```

Comment:
Shouldn't this be a geometry type instead of json?

https://hasura.io/blog/native-support-for-postgis-topology-operators-now-in-graphql-engine/

Reply:
IIRC, you can insert geometry objects as GeoJSON, isn't that right? But if we are to stay with json I would prefer we stick with jsonb.

Reply:
Swapping this out for jsonb was straightforward - looking forward to see the next phase of all this!